### PR TITLE
Docs: Added meaningful comments to ellipses in installing plugins guide.

### PR DESCRIPTION
**Pull request by ldrobnik**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Docs: Added meaningful comments to ellipses in installing plugins guide. Closes [#13039](https://github.com/ckeditor/ckeditor5/issues/13039).

---

### Additional information

Added explanatory comments to ellipses in a code snippet in the installing plugins guide.

### CODE REVIEW

File: docs/installation/getting-started/installing-plugins.md [hunk 1]
```diff
@@ -243,6 +244,7 @@ ClassicEditor.builtinPlugins = [
 	Bold,
 	Italic,
 	BlockQuote,
+	// Here you should add all the other plugins you want to include in the build.
```
[the remark this answers is not in the document]
Comment (dufipl) [reply docs/installation/getting-started/installing-plugins.md:247]: Same as above ☝️, the user is rather not expected to modify something here in this guide, so we inform him what would be here in the full version of the file using almost the same words as in the original source: https://github.com/ckeditor/ckeditor5-build-classic/blob/0501682eced14cb7d362d8f21228be852cd90a92/src/ckeditor.js#L35

File: docs/installation/getting-started/installing-plugins.md [hunk 2]
```diff
@@ -251,9 +253,11 @@ ClassicEditor.defaultConfig = {
 		items: [
 			'heading',
 			'bold',
+			// Add the remaining toolbar items here.
```
Comment (dufipl) [inline docs/installation/getting-started/installing-plugins.md:256]:
```suggestion
			// Remaining toolbar items.
```

Comment (dufipl) [reply docs/installation/getting-started/installing-plugins.md:256]: As above comments ☝️

File: docs/installation/getting-started/installing-plugins.md [hunk 3]
```diff
@@ -251,9 +253,11 @@ ClassicEditor.defaultConfig = {
...
 			// ...
 		]
 	},
+	// Specify additional configuration here.
```
Comment (dufipl) [inline docs/installation/getting-started/installing-plugins.md:260]:
```suggestion
	// Rest of classic editor configuration. 
```

Comment (dufipl) [reply docs/installation/getting-started/installing-plugins.md:260]: As above comments ☝️

**Review by dufipl — CHANGES_REQUESTED** [review]
I've prepared inline suggestions that, if you agree, you may commit directly from the GitHub. They are based on the fact, that this particular example of code is just used to explain something and is based on the existing code of `ckeditor5-build-classic` (https://github.com/ckeditor/ckeditor5-build-classic/blob/stable/src/ckeditor.js) so the user is not expected to copy it or change anything within. Consequently, we should just describe what is in these places of files where ellipses were used (as they were used in order to shorten the code example).

File: docs/installation/getting-started/installing-plugins.md [hunk 4]
```diff
@@ -232,6 +232,7 @@ import Autoformat from '@ckeditor/ckeditor5-autoformat/src/autoformat';
 import Bold from '@ckeditor/ckeditor5-basic-styles/src/bold';
 import Italic from '@ckeditor/ckeditor5-basic-styles/src/italic';
 import BlockQuote from '@ckeditor/ckeditor5-block-quote/src/blockquote';
+// Add the remaining imports here.
```
[the remark this answers is not in the document]
Comment (ldrobnik) [reply docs/installation/getting-started/installing-plugins.md:235]: Much better. Thanks for this. I was wondering what style I should choose.

File: docs/installation/getting-started/installing-plugins.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ldrobnik) [reply docs/installation/getting-started/installing-plugins.md:247]: Your suggestion makes much more sense. Many thanks!

File: docs/installation/getting-started/installing-plugins.md (hunk 2 above)
Comment (ldrobnik) [reply docs/installation/getting-started/installing-plugins.md:256]: Thank you!

File: docs/installation/getting-started/installing-plugins.md (hunk 3 above)
Comment (ldrobnik) [reply docs/installation/getting-started/installing-plugins.md:260]: Many thanks!

**Review by dufipl — APPROVED** [review]
LGTM 🎉

**Review by dufipl — CHANGES_REQUESTED** [review]
Unfortunately, now I see that PR is set to be merged to 'master', whereas it should be merged to 'stable'. We should close it and create another one.

**Comment by ldrobnik** [comment]
Oh no, my bad. Sorry! I'll create a new one.

**Comment by dufipl** [comment]
Closing as the wrong branch is set to be merged into. @ldrobnik please create a new one in order to merge from your branch to the `stable` and add me to the final review :)

**Pull request closed without merging**